src/views/Story.vue

```diff
@@ -1,21 +1,32 @@
 <template>
 <div class="view story">
+    <PrimaryButton class="publish-button" :text="$i18n('btn-publish')" :onClick="onPublish" />
```

Comment:
I think we're supposed to show the publish button only in the last frame, from what I can gather from the figma

Reply:
Thanks. just check the design, yes you are right! changed. https://github.com/wikimedia/wikistories-prototype/pull/16/commits/613be16739e0a37f611a414eaa56f0e75189231b